### PR TITLE
Iteration1 1

### PR DESCRIPTION
**Pull request by mike-duke**
i implemented the scrolling text from my code pen into the ScrollingText component and i have the random crawl working. it prints the episode number instead of a roman numeral, but i like that better than not having the episode at all. According to the spec, we're supposed to have the release date of the film on the scroll also, but we can add that easily. Finally, i noticed that there is a small delay getting the data on refresh that causes the scroll to display as undefined, so we should probably deal with that at some point

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -28,9 +34,13 @@ class App extends Component {
   }
 
   render() {
+    const { crawl, title, episode } = this.state.randomCrawl;
+    console.log(this.state)
```
Comment (Dhanciles) [inline src/Components/App/App.js]: I would delete this console.log as well

**Comment by Cody-Price** [comment]
Hey mike! love the scrolling text it looks just like the movie! On a technical note i did notice a few things: 
    there is a console.log() on line 18 and line 38
    I did notice that we are using Math.random() multiplied by 7 however I dont think Math.random() ever reaches a perfect 1 so i think we should change it to multiplied by 8
    last thing i noticed was that our ScrollingText component is using an ES5 syntax however just by convention i think we should possibly make it an ES6 arrow function.
thoughts?
once again great job!

File: src/Components/App/App.js [hunk 2]
```diff
@@ -28,9 +34,13 @@ class App extends Component {
   }
 
   render() {
```
Comment (Dhanciles) [inline src/Components/App/App.js]: I think maybe having a loading component would help with the issues with compenentDidMount giving us our default state when our component has not finished mounting.

**Comment by Dhanciles** [comment]
Great work on this Mike! This all looks really good. Once we have fixed the comments that me and Cody have made I think we are good to merge this in!

**Comment by mike-duke** [comment]
i want to leave the console.log on line 18 for now so that we can continue to see the random scroll as we are working. we will definitely need to remove it before the eval tho. Thanks!

**Pull request merged**